.travis.yml

```diff
@@ -9,7 +9,7 @@ matrix:
       script:
         - ./make/manylinux1/build_wheels.sh
     - os: osx
-      osx_image: xcode9.4
+      osx_image: xcode8.3
```

Comment:
Was there a reason we downgraded the xcode version here?

Reply:
I think it was something Moisés recommended because of the customer.